samples/ffmpeg-wasm-demuxer/blocking_demuxer_worker.js

```diff
@@ -0,0 +1,64 @@
+// The "blocking demuxer worker" is where FFmpeg demuxing actually takes place.
+// Blocking occurs via Atomics.wait() within FFmpegDemuxerBlockingHelper.
+// Blocking is required because FFmpeg's AVIO read callback is synchronous,
+// while the data-to-be-read is being asynchronously fetched from the network.
+
+self.debugLog = function(msg) {
+    console.debug('[blocking worker]' + msg);
+}
+
+debugLog(` -- worker started`);
```

Comment:
Nit: ` -- ` not required with above change.
Nit: Template string literal not necessary for static string.